tests/codemod/deprecate-version/test-data.ts

```diff
@@ -0,0 +1,35 @@
+export const inputCode = `
+import { Color } from "@lifesg/react-design-system/color";
+import { MediaQuery } from "@lifesg/react-design-system";
+import { Text } from "@lifesg/react-design-system/text";
+import {DesignToken} from "../design-token";
+
+const Component = () => (
+    <>
+        <DesignToken />
+        <Color />
+        <MediaQuery />
+        <Text />
```

Comment:
a more realistic example would be

```suggestion
        <Text.Body />
```